main.py

```diff
@@ -1,10 +1,59 @@
+import json
+
 import init_django_orm  # noqa: F401
 
-from db.models import Race, Skill, Player, Guild
+from db.models import Race, Guild, Skill, Player # noqa
+
+
+def get_or_create_race(race: dict) -> Race:
+    obj, _ = Race.objects.get_or_create(
+        name=race.get("name"),
+        description=race.get("description")
+    )
+    return obj
+
+
+def get_or_create_guild(guild: dict | None) -> Guild | None:
+    if guild is None:
+        return
+    obj, _ = Guild.objects.get_or_create(
+        name=guild.get("name"),
+        description=guild.get("description")
+    )
+    return obj
+
+
+def add_player(
+        player: dict,
+        nickname: str,
+        race: Race,
+        guild: Guild
+) -> Player:
+    obj, _ = race.player_set.get_or_create(
+        nickname=nickname,
+        email=player.get("email"),
+        bio=player.get("bio"),
+        guild=guild
```

Comment:
Ensure that `guild` is not None before using it here. Since `guild` can be None, you might want to handle this case explicitly to avoid potential errors.